docs/csharp/fundamentals/functional/deconstruct.md

```diff
@@ -0,0 +1,103 @@
+---
+title: Deconstructing tuples and other types
+description: Learn how to deconstruct tuples and other types.
+ms.technology: csharp-fundamentals
```

Comment:
Some of the other articles moved ms.tech. Double check things will be caught by the global rules, if they are to be set by global rules, why is this being set?

Reply:
Good catch. They should all be caught by global rules. I'll remove those that aren't.